src/index.css

```diff
@@ -1,12 +1,12 @@
 :root {
-  --main-color: #8aba87;
+  --main-color: #f4f4f4;
   --anti-color: #b788ba;
 }
 
 body {
   font-family: system, -apple-system, ".SFNSText-Regular", "SF UI Text",
     "Lucida Grande", "Segoe UI", Ubuntu, Cantarell, sans-serif;
-  color: #fff;
+  color: rgb(0, 0, 0);
   background-color: var(--main-color);
   text-align: center;
   font-size: 40px;
```

Comment:
The problem with styling standard elements like `body` or `h1` is, that they can be styled everywhere and in places you might not want it to be styled that way. For instance here `font-size: 40px` is valid everywhere inside the body (where it is not overwritten). I recommend using classes and not styling the regular selectors.

Reply:
i didnt create the body changes. :) we can delete them completly if everyone is ok with it. they were created in the index.css

Reply:
*I didnt create the body class ;)
 I did the changes

Reply:
awesome 👍